llvm-external-projects/iree-compiler-api/CMakeLists.txt

```diff
@@ -42,7 +42,8 @@ else()
   set(LLVM_MAIN_SRC_DIR "${IREE_COMPILER_API_SOURCE_DIR}/../../third_party/llvm-project/llvm")
   set(LLVM_MAIN_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/llvm")
   set(LLVM_EXTERNAL_MLIR_HLO_SOURCE_DIR "${IREE_COMPILER_API_SOURCE_DIR}/../../third_party/mlir-hlo")
-  if(${IREE_BUILD_TORCH_MLIR_SUPPORT})
+  option(IREE_BUILD_TORCH_MLIR_SUPPORT "Builds support for compiling torch-mlir programs." ON)
```

Comment:
Is this one special here compared to EmitC and the other options in https://github.com/google/iree/blob/75fa2e7c8be3dfb3c5c1ac08d5ae5c1708a3e42f/CMakeLists.txt#L60 as those are  off by default?

Reply:
yea, changing it to default OFF would be more consistent with other experimental projects. This would require either adding a new CI test with the option ON or turning on the option on an existing test. Will need suggestions from people who are more familiar with IREE infra to chime in.

Reply:
To clarify, the question is at 2 levels, first one is that whether we want to: 
1. make `IREE_BUILD_TORCH_MLIR_SUPPORT` default OFF as in this PR  https://github.com/google/iree/pull/8896 
or
2. keep IREE_BUILD_TORCH_MLIR_SUPPORT` default ON and duplicate the `IREE_BUILD_TORCH_MLIR_SUPPORT` option before this line https://github.com/google/iree/blob/abe06bbd9637c663637694c19f5d26397b7a1ca7/llvm-external-projects/iree-compiler-api/CMakeLists.txt#L45. 

If we would like to move forward with 1, whether we should add a new CI test with the option ON or turning on the option on an existing test.

Reply:
Let's do 2 for now. Smallest change that enablee standalone build to work. Please file an issue or add TODO to ensure we get back to this.

Reply:
Just saw this message. OK, I will go ahead with 2. Opened an issue https://github.com/google/iree/issues/8906 to wait for @ScottTodd 's input.